Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -854,11 +854,15 @@ protected class ClassWriter : IClassWriter {
 
     protected override void EmitCollectionDisplay(CollectionType ct, IToken tok, List<Expression> elements,
       bool inLetExprBody, ConcreteSyntaxTree wr, ConcreteSyntaxTree wStmts) {
+      var (open, close) = ct switch {
+        SeqType => ("[", "]"),
+        _ => ("{", "}")
+      };
```

Comment:
Ideally we'd add a test for this as well if we're going to implement it, even though I'm pretty darn sure this is correct. :) I'm cool with this since I'm confident we'll pull in more test cases with sequence displays very soon.